bindata/network/ovn-kubernetes/self-hosted/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -746,162 +404,8 @@ spec:
-          fi
-          if [[ -n "${OVNKUBE_NODE_MGMT_PORT_DP_RESOURCE_NAME}" ]] ; then
-            node_mgmt_port_netdev_flags="$node_mgmt_port_netdev_flags --ovnkube-node-mgmt-port-dp-resource-name ${OVNKUBE_NODE_MGMT_PORT_DP_RESOURCE_NAME}"
-          fi
-
-          multi_network_enabled_flag=
-          if [[ "{{.OVN_MULTI_NETWORK_ENABLE}}" == "true" ]]; then
-            multi_network_enabled_flag="--enable-multi-network"
-          fi
-
-          multi_network_policy_enabled_flag=
-          if [[ "{{.OVN_MULTI_NETWORK_POLICY_ENABLE}}" == "true" ]]; then
-            multi_network_policy_enabled_flag="--enable-multi-networkpolicy"
-          fi
-
-          admin_network_policy_enabled_flag=
-          if [[ "{{.OVN_ADMIN_NETWORK_POLICY_ENABLE}}" == "true" ]]; then
-            admin_network_policy_enabled_flag="--enable-admin-network-policy"
-          fi
-
-          # If IP Forwarding mode is global set it in the host here.
-          ip_forwarding_flag=
-          if [ "{{.IP_FORWARDING_MODE}}" == "Global" ]; then
-            sysctl -w net.ipv4.ip_forward=1
-            sysctl -w net.ipv6.conf.all.forwarding=1
-          else
-            ip_forwarding_flag="--disable-forwarding"
-          fi
-
-          NETWORK_NODE_IDENTITY_ENABLE=
-          if [[ "{{.NETWORK_NODE_IDENTITY_ENABLE}}" == "true" ]]; then
-            NETWORK_NODE_IDENTITY_ENABLE="
-              --bootstrap-kubeconfig=/var/lib/kubelet/kubeconfig
-              --cert-dir=/etc/ovn/ovnkube-node-certs
-              --cert-duration={{.NodeIdentityCertDuration}}
-            "
-          fi
-
-          exec /usr/bin/ovnkube --init-ovnkube-controller "${K8S_NODE}" --init-node "${K8S_NODE}" \
-            --config-file=/run/ovnkube-config/ovnkube.conf \
-            --ovn-empty-lb-events \
-            --loglevel "${OVN_KUBE_LOG_LEVEL}" \
-            --inactivity-probe="${OVN_CONTROLLER_INACTIVITY_PROBE}" \
-            ${gateway_mode_flags} \
-            ${node_mgmt_port_netdev_flags} \
-            {{- if eq .OVN_NODE_MODE "dpu-host" }}
-            --ovnkube-node-mode dpu-host \
-            {{- end }}
-            --metrics-bind-address "127.0.0.1:29103" \
-            --ovn-metrics-bind-address "127.0.0.1:29105" \
-            --metrics-enable-pprof \
-            --metrics-enable-config-duration \
-            --export-ovs-metrics \
-            --disable-snat-multiple-gws \
-            ${export_network_flows_flags} \
-            ${multi_network_enabled_flag} \
-            ${multi_network_policy_enabled_flag} \
-            ${admin_network_policy_enabled_flag} \
-            --enable-multicast \
-            --zone ${K8S_NODE} \
-            --enable-interconnect \
-            --acl-logging-rate-limit "{{.OVNPolicyAuditRateLimit}}" \
-            ${gw_interface_flag} \
-            --enable-multi-external-gateway=true \
-            ${ip_forwarding_flag} \
-            ${NETWORK_NODE_IDENTITY_ENABLE}
+          . /ovnkube-lib/ovnkube-lib.sh || exit 1
```

Comment:
K8S_NODE sourced above is aslo sourced within ovnkube-lib.sh

Reply:
Fixed.